frontend/src/components/BaseButton.vue

```diff
@@ -1,7 +1,6 @@
 <template>
   <div>
     <q-btn
-      class="q-my-sm"
```

Comment:
For 1. I assumed you meant remove cursor-pointer which I did  along with 2 and 3.

Reply:
Oh sorry, I can see how that phrasing was unclear. I meant on this highlighted portion + the avatar to it's right 
<img width="173" alt="image" src="https://user-images.githubusercontent.com/17163988/199350790-b122ea54-faea-4474-8fec-eaa440262005.png">

Though I'm now also realizing that we're showing a jazzicon instead of ENS avatar in the modal, right above "copy adress".

One other nit is, let's move the ENS avatar to the left of the ENS name, and the wizard icon on the right. That feels more consistent, since then the network selector + both ENS names shown all have logo on the left and text to the right

Reply:
I added these changes, but was unable to test an ENS image. Hopefully it is working if not should be a small change

Reply:
super close! I think you just did the advanced mode icon instead of the ENS avatar (IMO we don't need the advanced mode icon in this pop up modal)

Reply:
so to clarify based on the above image, I'd say what's left is 
- replace the advanced mode icon (the one directly below the word "Connected") with ENS avatar that falls back to jazzicon (i.e. the same as we do for the avatar that's directly to the left of the network selector)
- then, in the top row we have `advanced mode icon | ENS name | ENS avatar`, but let's flip that too `ENS avatar | ENS name | advanced mode icon`. I think this ordering is more common in other apps + it matches what's in the pop up modal which looks nice

(sorry for being nitpicky 🙈)

Reply:
Okay, great that makes sense, sorry for the confusion. Is there a easy way to test ENS avatar as I don't have one setup on my test wallets.

Reply:
You can use https://www.impersonator.xyz/ with my ENS above. Obviously you can’t send txs from that site but you can do everything else as if you were any address

Reply:
I think it should be correct now, and I tested with impersonator thanks for the recommendation.

![Screenshot from 2022-11-05 10-48-25](https://user-images.githubusercontent.com/12705146/200125819-8c9f0d19-14ac-4d89-9eab-c4eae8b980b6.png)

Reply:
One other small thing I also just noticed is that disconnect doesn't clear lastWallet which means it will autoconnect to the last wallet a user used like metamask. Should we also clear lastWallet on disconnect or handle that in a separate ticket?

Reply:
Nice, yea this is all looking great now, thank you! I think it's ok that we don't clear lastWallet:
- If they connect a new wallet after disconnecting (which I'd guess is the main use case), that will become the last wallet
- If they don't connect a new wallet, then it will auto-reconnect next time, but I just tested on uniswap and they behave the same way, so it's probably ok that we also do that

(Leaving this comment unresolved to ensure you see the reply, feel free to resolve this comment yourself)